### PR TITLE
Fixed spacing in method call.

### PR DESCRIPTION
**Pull request by lothrop**
## Summary

Fixes non-standard spacing in the code snippet.

### CODE REVIEW

File: snippets/csharp/VS_Snippets_Data/XmlReader.Create/CS/XmlReader_Create.cs [hunk 1]
```diff
@@ -47,7 +47,7 @@ static void Settings_Resolver()
 // Create an XmlUrlResolver with the credentials necessary to access the Web server.
 XmlUrlResolver resolver = new XmlUrlResolver();
 System.Net.NetworkCredential myCred;
```
Comment (Youssef1313) [inline snippets/csharp/VS_Snippets_Data/XmlReader.Create/CS/XmlReader_Create.cs]:
```suggestion
```

File: snippets/csharp/VS_Snippets_Data/XmlReader.Create/CS/XmlReader_Create.cs [hunk 2]
```diff
@@ -47,7 +47,7 @@ static void Settings_Resolver()
...
-myCred  = new System.Net.NetworkCredential(UserName,SecurelyStoredPassword,Domain);  
+myCred = new System.Net.NetworkCredential(UserName, SecurelyStoredPassword, Domain);
```
Comment (Youssef1313) [inline snippets/csharp/VS_Snippets_Data/XmlReader.Create/CS/XmlReader_Create.cs]:
```suggestion
var myCred = new NetworkCredential(UserName, SecurelyStoredPassword, Domain);  
```

**Comment by lothrop** [comment]
It feels awkward to introduce `var` in one line and not have it in the line above (where the type is also obvious). Should I also change this?

**Comment by mairaw** [comment]
Yes please!

**Comment by NextTurn** [comment]
@BillWagner What are the coding standards you referred to? Are they documented somewhere?

**Comment by BillWagner** [comment]
@NextTurn We generally follow the [dotnet/runtime coding standard](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md).

"Generally", because some samples are meant to highlight libraries that aren't part of those standards. Also, we'll prefer a better reading experience over production coding practices when those disagree.

**Pull request merged**